### ISSUE TITLE
 Applied Accessibility: Improve Accessibility of Audio Content with the audio Element - says there should be  closing tag even though there already is one

### ISSUE THREAD

**Issue by AaronWands** [freeCodeCamp/freeCodeCamp#17729]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/improve-accessibility-of-audio-content-with-the-audio-element
#### Describe your problem and - if possible - how to reproduce it
It says there should be a closing tag even though a closing tag is already there

#### Add a Link to the page with the problem
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/improve-accessibility-of-audio-content-with-the-audio-element

#### Tell us about your browser and operating system

* Browser Name:  Chrome
* Browser Version:  No idea
* Operating System: Windows 10
